Src/Clients/WebUI/wwwroot/css/site.css

```diff
@@ -47,7 +47,7 @@ html {
 }
 
 body {
-    background-image: repeating-linear-gradient(#65d3d3, #eae5e5, #00ffff);
+    /*background-image: repeating-linear-gradient(#65d3d3, #eae5e5, #00ffff);*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)